src/script/main/app.js

```diff
@@ -598,7 +602,8 @@ class App {
    * @returns {undefined} No return value
    */
   _showInterface() {
-    const mainView = new z.viewModel.MainViewModel(this.repository);
+    const mainView = new MainViewModel(this.repository);
+    ko.applyBindings(mainView, this.domContainer);
```

Comment:
Now, only the App class knows where to bootstrap the application.
The MainViewModel has no idea about the DOM structure anymore.